#goal
uint8 BROW_RELAXED = 0
uint8 BROW_FURROW = 1
uint8 RAISED_INNER_BROW = 2

uint8 browMovementType

uint8 DEGREE_LOW = 0
uint8 DEGREE_MEDIUM = 1
uint8 DEGREE_HIGH = 2
uint8 degree_of_movement
duration gesture_duration

---
#feedback

uint8 MOVING_TO_TARGET = 0
uint8 AT_TARGET = 1
uint8 feedbackState
---
#state
